resources/model/OpenStudio.idd

```diff
@@ -26403,7 +26403,6 @@ OS:ZoneHVAC:TerminalUnit:VariableRefrigerantFlow,
         \note This field is also ignored if VRF terminal unit is used on main AirloopHVAC branch
         \note and terminal unit has no fan.
   A8 ,  \field Supply Air Fan
-        \required-field
```

Comment:
So, this is not required anymore... I have not changed the public model API for now because I would had to break backward compatibility. @kbenne